var/spack/repos/builtin/packages/elfutils/package.py

```diff
@@ -89,13 +89,10 @@ class Elfutils(AutotoolsPackage, SourcewarePackage):
     # and https://github.com/libarchive/libarchive/issues/1819
     conflicts("^libarchive@3.6.2 +iconv", when="+debuginfod")
 
-    # Elfutils uses nested functions in C code, which is implemented
-    # in gcc, but not in clang. C code compiled with gcc is
-    # binary-compatible with clang, so it should be possible to build
-    # elfutils with gcc, and then link it to clang-built libraries.
-    conflicts("%apple-clang")
-    conflicts("%clang")
-    conflicts("%cce")
+    # https://sourceware.org/bugzilla/show_bug.cgi?id=24964
+    conflicts("%apple-clang", when="@:0.186")
```

Comment:
The version range here is inclusive. The way this is written thus means the conflict exists for `elfutils@0.186` -- is this what the intent was?

Reply:
That was not my intention. I've updated the patch to change the conflict to only go up to `0.185` since clang support was added in `0.186`. Thanks for the catch!